elrond-wasm/src/api/send_api.rs

```diff
@@ -39,8 +39,8 @@ where
 		token: &[u8],
 		amount: &BigUint,
 		data: &[u8],
-	) {
-		self.direct_esdt_execute(to, token, amount, 0, data, &ArgBuffer::new());
+	) -> i32 {
```

Comment:
This is a hack, we probably shouldn't copy the pattern to all API methods, but I'll allow it for now.